tests/test_media_handlers.py

```diff
@@ -1,3 +1,6 @@
+import unittest.mock as mock
```

Comment:
Note: ``mock`` does not exist in the Python 2 world this way, needs to be pip-installed.